### ISSUE TITLE
Issue from ÉTS Mobile 

### ISSUE THREAD

**Issue by clubapplets-server** [ApplETS/Notre-Dame#177]
**Describe the issue** 
``````

**Screenshot** 
![screenshot](https://github.com/ApplETS/Notre-Dame-Bug-report/blob/main/bugPicture-317954566.png?raw=true)

**Device Infos** 
- **Version:** 4.2.2 
- **Connectivity:** ConnectivityResult.wifi 
- **Build number:** 1630444374 
- **Platform operating system:** android 
- **Platform operating system version:** CPH2005_11_C.83

**Comment by apomalyn** [comment]
Mh I've tested it this morning (the `love you guys`) and it was working.
It seems the picture wasn't uploaded

**Comment by apomalyn** [comment]
@AlexandreMarkus found the problem, well at least a part of it. If the user doesn't add any description the output format is broken.

**Comment by AlexandreMarkus** [comment]
@apomalyn we could simply add a check, is the string is empty, add "no description available" text to fix the problem

**Comment by apomalyn** [comment]
Yeah, I think I would prefer to ask for a description ahah ^^' This will have to wait the auto-deploy PR

**Comment by AlexandreMarkus** [comment]
> Yeah, I think I would prefer to ask for a description ahah ^^' This will have to wait the auto-deploy PR

Yeah sure!!!, It's better that way ^^

**Comment by apomalyn** [comment]
@ApplETS/general what do you think about giving a way for the user to see if his issue is treated ?

I see two options (if you have an idea comment them):

1. After the submit we give the user the id of the issue (very easy to implement)
2. Add a view where he/she can see the issues he/she created.

Vote using 1️⃣ or 2️⃣  😄

**Comment by MysticFragilist** [comment]
@apomalyn That could be really useful indeed! Like the last issue #208 I think it's only the person that didn't do his update on the next version. He should receive some kind of feedback to know that there is a new update out there.

**Comment by apomalyn** [comment]
@MysticFragilist sorry for the answering time 😅 
I agree with you. For now, I suggest doing the following things:

- [ ] Make the description field in the feedback required
- [ ] Add the universal code to the device info so we could give it to IT if needed.
- [ ] Ask the user email to contact him if necessary
- [ ] Add a page 'Current issues' or 'Known issues' in the more view that open a web view with all the current issues OR only the issues created by the user (personally I prefer the first option)

What do you think about this? 😄

**Comment by camillebrulotte** [comment]
Coucou @apomalyn  ! :D Je suis entrain de faire cette issue, par contre pour le code universel je suis pas certaine que ce soit trop legit de le mettre 'public' dans l'issue à moins que tu le voyais d'une autre manière ? :o J'ai mis le courriel (facultatif) par contre donc on pourrait simplement leur demander leur code uni par courriel si jamais ! 

Et j'ai refais un peu le template si jamais vous avez des améliorations à apporter d'ici là : 
<img src="https://user-images.githubusercontent.com/37625944/186431645-e173eb41-b57d-4041-84be-3a0bcc2c6d0d.png" height="600">
<img src="https://user-images.githubusercontent.com/37625944/186432056-71ecc83c-3669-4218-b687-d30efe2c1bde.png" height="600">

**Comment by AlexandreMarkus** [comment]
> Coucou @apomalyn  ! :D Je suis entrain de faire cette issue, par contre pour le code universel je suis pas certaine que ce soit trop legit de le mettre 'public' dans l'issue à moins que tu le voyais d'une autre manière ? :o J'ai mis le courriel (facultatif) par contre donc on pourrait simplement leur demander leur code uni par courriel si jamais ! 
> 
> Et j'ai refais un peu le template si jamais vous avez des améliorations à apporter d'ici là : 
> <img src="https://user-images.githubusercontent.com/37625944/186431645-e173eb41-b57d-4041-84be-3a0bcc2c6d0d.png" height="600">
> <img src="https://user-images.githubusercontent.com/37625944/186432056-71ecc83c-3669-4218-b687-d30efe2c1bde.png" height="600">

Wow🤩 j'adore ce nouveau design!!

**Comment by apomalyn** [comment]
@camillebrulotte déjà bravo pour le nouveau design! J'adore aussi! 

Go pour mettre le courriel uniquement, c'est vrai que mettre le code universel pourrait être mal vu. L'idée du courriel est plus propre! ^^ GG
